metagraph/src/graph/representation/succinct/boss.hpp

```diff
@@ -283,10 +283,10 @@ class BOSS {
      * Representation states of the BOSS table.
      *
      * SMALL is the smallest
-     * STAT provides a good space/time tradeoff
+     * STAT provides the best space/time trade-off
```

Comment:
why not say "the best (at the time of writing)" to make the statement less strong? :)

Reply:
Strong is fine here, it's not a theorem, it's not even defined what this tarde-off is.

If at some point some better representations will be developed, this state should be updated and we will switch to those better representations.